Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -7,7 +7,10 @@
 //
 
 import SDL
+
+#if os(Android)
 import JNI
+#endif
```

Comment:
I think we can remove this since we're already importing JNI in the same file below?